README.md

```diff
@@ -2,6 +2,33 @@
 
 Formal and executable specifications for the new features to be introduced by Shelley.
 
+The documents are built in our CI and can be readily accessed using the
+following links:
+
+- [Delegation Design Specification](https://hydra.iohk.io/job/Cardano/fm-ledger-rules/delegationDesignSpec/latest/download-by-type/doc-pdf/delegation_design_spec)
+- [Shelley specification](https://hydra.iohk.io/job/Cardano/fm-ledger-rules/shelleyLedgerSpec/latest/download-by-type/doc-pdf/ledger-spec)
+- [Non-integer Calculations Specification](https://hydra.iohk.io/job/Cardano/fm-ledger-rules/nonIntegerCalculations/latest/download-by-type/doc-pdf/non-integer-calculations)
+- [Byron Chain Specification](https://hydra.iohk.io/job/Cardano/fm-ledger-rules/byronChainSpec/latest/download-by-type/doc-pdf/blockchain-spec)
+- [Byron Ledger Specification](https://hydra.iohk.io/job/Cardano/fm-ledger-rules/byronLedgerSpec/latest/download-by-type/doc-pdf/ledger-spec)
+- [Small-step-semantics Framework Explanation](https://hydra.iohk.io/job/Cardano/fm-ledger-rules/semanticsSpec/latest/download-by-type/doc-pdf/semantics-spec)
+
+## Build tools
+
+For building LaTeX documents we use
+[`nix`](https://nixos.org/nix/download.html). Haskell files can be built either
+with `nix` or [`stack`](https://docs.haskellstack.org/en/stable/README/).
+
+When using `nix` it is recommended that you setup the cache, so that it can
+reuse built artifacts, reducing the compilation times dramatically:
+
+Put the following in `/etc/nix/nix.conf` if you have a system-wide `nix`
+installation , or in `~/.config/nix/nix.conf` if you have a local installation:
+
+```nix
+substituters        = https://hydra.iohk.io https://cache.nixos.org/
+trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
+```
```

Comment:
Should add in the nixos configuration, or do we just assume that nixos users will know how to do this?

Reply:
I think we should be as user friendly as possible, so if there are some non-obvious pieces of information missing we should add them. But in this case I'm not sure what should we add :) Suggestions?